packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -77,7 +77,7 @@ class CollapsibleMenuViewItemGroup extends React.Component {
       );
     } if (isCollapsibleMenuItem) {
       return (
-        <List {...customProps} onChange={this.handleOnChange}>
+        <List {...customProps} role="listbox">
```

Comment:
Did we need to pass the onChange down still?

Reply:
No, I discussed this wish Jaime, the onChange was added uncessarily and the list functions with the onClicks of each menuItem.